### PR TITLE
Likes/Appreciations 

### PR DESCRIPTION
**Pull request by jamesmorgan**
* [x] Hooking up to API 
* [x] Simple component showing total and if you've liked it
* [x] Like button component showing total and if you've liked it

**DONT MERGE YET**

### CODE REVIEW

File: src/components/pages/Account.vue [hunk 1]
```diff
@@ -55,24 +85,50 @@
         </div>
       </div>
 
+      <div class="row editions-wrap" v-if="accountFavorites">
+
+        <div class="col-12">
+          <h5 class="mb-3">Your Favorites</h5>
```
Comment (andygray) [inline src/components/pages/Account.vue]: 👍

File: src/components/pages/Account.vue [hunk 2]
```diff
@@ -55,24 +85,50 @@
...
+               :key="editionNumber"
+               v-if="assets[editionNumber] && assets[editionNumber].active">
+            <gallery-card :edition="assets[editionNumber]" :edition-number="editionNumber">
+            </gallery-card>
+          </div>
+        </div>
+
+        <div class="row editions-wrap pt-1 pb-4" v-if="canShowMoreLikes">
+          <div class="col-12 text-center">
+            <button @click="showMoreLikes" class="btn btn-lg btn-outline-primary">Show more</button>
```
Comment (andygray) [inline src/components/pages/Account.vue]: use `btn-block` and drop `btn-lg` to match gallery?

File: src/components/pages/Account.vue [hunk 3]
```diff
@@ -87,47 +143,100 @@
...
+        loadFavs();
       }.bind(this);
 
-      this.$store.watch(
-        () => this.$store.state.account,
-        () => loadData()
-      );
+      const loadFavs = () => {
+        if (this.$store.state.account && this.$store.state.currentNetworkId) {
+          axios.get(`${getApi()}/likes/address/${this.account}?network=${this.currentNetworkId}`)
```
Comment (andygray) [inline src/components/pages/Account.vue]: why not pull direct from the firestore? Just wondering...

File: src/components/pages/Account.vue (hunk 2 above)
Comment (jamesmorgan) [reply src/components/pages/Account.vue]: changed

File: src/components/pages/Account.vue (hunk 3 above)
Comment (jamesmorgan) [reply src/components/pages/Account.vue]: Hiding it behind a API hides some of the complexities of the data store and means we can change the backend without the frontend know about it, the contract API stays the same.

**Pull request merged**